Directory.Packages.props

```diff
@@ -1,7 +1,9 @@
 <Project>
+
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <OTelLatestStableVer>1.9.0</OTelLatestStableVer>
+    <LatestRuntimeOutOfBandVer>9.0.0-rc.1.24431.7</LatestRuntimeOutOfBandVer>
```

Comment:
nit: Are you planning on renaming "OutOfBandVer" when .NET9 is GA?

I fully support parameterizing this value for ease of maintenance, just curious about the name.

Reply:
I used "OutOfBand" here because that's what runtime calls its libraries which are pushed to NuGet: https://github.com/dotnet/runtime/blob/main/src/libraries/README.md#deployment

Happy to rename it if you think something else would be better though.

Reply:
Ended up removing this from the PR. I'll bring it back as a follow-up.